inputfiles/browser.webidl.xml

```diff
@@ -3241,7 +3241,10 @@
         <method name="createHTMLDocument" type="Document">
           <param name="title" type="DOMString" />
         </method>
-        <method name="hasFeature" type="boolean" />
+        <method name="hasFeature" type="boolean">
```

Comment:
@mhegazy Thanks! Sorry about that, should have read the README more closely!

I have reverted the changes to the xml files and added an override.